### PR TITLE
Modestly efficient is_prime function, with edge-case testing

### PR DESCRIPTION
**Pull request by donalgrant**
is_prime function added to simple_functions, which tries to short-cut easy cases, and steps through every odd integer (up to sqrt(x)) to look for factors of the argument.

Added test function for is_prime to simple_test_function with assertions for several edge cases.

### CODE REVIEW

File: participants/donalgrant/pr_tutorial/tests/test_simple_function.py [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+
+def test_is_prime():
+    '''Example of composite number'''
+    assert is_prime(1) == False
+    assert is_prime(2) == True
+    assert is_prime(3) == True
+    assert is_prime(4) == False
+    assert is_prime(49) == False
+    assert is_prime(73) == True
+    assert is_prime(459) == False
```
Comment (ketozhang) [inline participants/donalgrant/pr_tutorial/tests/test_simple_function.py:18]: question: Remind me, are there any negative prime numbers? Probably opening a can of worms we don't need.

**Review by ketozhang — APPROVED** [review]
Looks good to me, just left a question.

File: participants/donalgrant/pr_tutorial/tests/test_simple_function.py (hunk 1 above)
Comment (donalgrant) [reply participants/donalgrant/pr_tutorial/tests/test_simple_function.py:18]: Interesting question, but I don't think so.  Prime Numbers are defined to be greater than 1.
